proxy/httpp/interface.go

```diff
@@ -182,6 +183,28 @@ func (t *Transform) Apply(req *http.Request) bool {
 		req.Header.Set("X-Webauth-Globalname", creds.Identity.GlobalName())
 	}
 
+	userGroups := map[string]struct{}{}
+	creds := oauth.GetCredentials(req.Context())
+	if creds != nil {
+		userGroups = slice.ToSet(creds.Identity.Groups)
+	}
+
+nextHeader:
+	for _, header := range t.MapRequestHeadersByGroup {
+		for _, groupMap := range header.GroupMapping {
+			// Empty group means this header value should always apply
+			if groupMap.Group == "" {
+				req.Header.Set(header.Header, groupMap.Value)
+				continue nextHeader
```

Comment:
nit: simple `break` here? rather than using a label on the outer loop? Don't feel strongly.

Reply:
I think the continue reads a bit more explicitly/naturally, and is more likely to work (or get re-adjusted) if code is copy/pasted